src/frontend/main.go

```diff
@@ -158,16 +182,26 @@ func initStats(log logrus.FieldLogger, exporter *stackdriver.Exporter) {
 }
 
 func initTracing(log logrus.FieldLogger) {
+	// This is a demo app with low QPS. trace.AlwaysSample() is used here
```

Comment:
Hmm I recall we had this already in. I wonder at what point it got removed. Thanks for adding back.